src/server/routes/emailTemplates.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Router } from 'express';
+import { ResponseWithRequestState } from '@/server/models/Express';
+import { render } from 'mjml-react';
+
+import { ResetPassword } from '../../email/templates/ResetPassword/ResetPassword';
+import { ResetPasswordText } from '../../email/templates/ResetPassword/ResetPasswordText';
+import { Verify } from '../../email/templates/Verify/Verify';
+import { VerifyText } from '../../email/templates/Verify/VerifyText';
+
+const router = Router();
+
+const emailTemplateTypes = ['ResetPassword', 'Verify'] as const;
```

Comment:
urls should be considered case-sensitive 
```URLs in general are case-sensitive (with the exception of machine names). There may be URLs, or parts of URLs, where case doesn't matter, but identifying these may not be easy. Users should always consider that URLs are case-sensitive```
https://www.w3.org/TR/WD-html40-970708/htmlweb.html

I would make these types lower case if they are also being used to match on the inbound url.